public/styles/queue.css

```
@@ -0,0 +1,114 @@
+.song-queue {
+  position: absolute;
+  min-height: 278px;
+  padding: 1rem;
+  right: 29px;
+  justify-items: center;
+  max-height: 100%;
+  background: rgba(0, 0, 0, 0.1);
+  border: 2px solid rgba(255, 255, 255, 0.1);
+  box-sizing: border-box;
+  backdrop-filter: blur(80px);
+  border-radius: 24px;
+  width: 1px;
+  opacity: 0;
+  transition: all .75s ease;
+  max-height: 400px;
+}
+
+.song-queue.show {
+  opacity: 1;
+  width: 275px;
+  max-height: 400px;
+  min-height: 278px;
+}
+
+.queue-list {
+  display: flex;
+  padding: 5px;
+  position: relative;
+  flex-direction: column;
+  max-height: 300px;
+  overflow-y: scroll;
```

Auto works here and will add a scrollbar only when necessary.
```suggestion
  overflow-y: auto;
```